text/0000-overlapping_match.md

```diff
@@ -45,23 +45,24 @@ match val in {
 ```
 
 Benefits of this syntax:
-1. No new keywords need to be used. This is good thing since it means for a relatively small
-addition there would be no code breaks of existing code with this change.
-2. The `in` seems to imply that it sort of like an "iterator" of statements and will go through
-each of them in turn.
+1. Even though a new keyword has been made it will not break any code because Rust is a context
+sensitive language. And adding such a keyword increases the perceptual area of the new syntax
+so as to make it clear which type of match is being used.
+2. The word `fallthrough` is used because it implies that after a branch is finished then the
+control falls through to the check of the next branch.
 
```

Comment:
I think `fallthrough` implies that it will execute the statements / yield the expressions of the next match arm irrespective of whether the it's pattern holds or not, i.e: readers will expect C++ style `switch` behavior.

Instead I add `match many` to the 🚲 🏠